source/documentation/obligationsandreturns.html.md

```diff
@@ -305,52 +350,52 @@ Software must send the calculationId that matches the calculation the customer i
-For all changes to annual summary updates including Self-employment, FHL property business and non-FHL property business:
-* software will have to resubmit any changes to the summary totals for income source. The nature of this obligation means there is no need to create separate update periods
-* where a business resubmits an annual summary update, previous figures that have been submitted must be sent again as well as any additional information. A zero or null will overwrite previously provided information
+* 	when a business resubmits an update period, the software will have to use the trigger calculation endpoint and follow the same process as the submitting an update period process 
+* 	software will have to resubmit any changes to the summary totals for income source. The nature of this obligation means there is no need to create separate update periods
+* 	where a business resubmits an annual summary update, previous figures that have been submitted must be sent again as well as any additional information. A zero or null will overwrite previously provided information
 * the software will have to use the [trigger calculation](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-assessment-api/2.0#tax-calculations_trigger-a-tax-calculation_post_accordion) endpoint and follow the same process
 
-There are some key points for changing previously submitted updates:
+Note: 
+
+* any changes that are made before the customer has crystallised is not a formal amendment For all changes to annual summary updates including Self-employment, FHL property business and non-FHL property business:
+
+## There are some key points for changing previously submitted updates:
+
+*	changes to Periodic updates - the update period you are trying to change must match the original update period exactly or it will be rejected
+*	changes to Annual updates - all figures previously supplied must be provided again, a Zero or a Null will overwrite any previously submitted information.
 
-* changes to Periodic updates - the update period you are trying to change must match the original update period exactly or it will be rejected
-* changes to Annual updates - all figures previously supplied must be provided again, a Zero or a Null will overwrite any previously submitted information.
 
 ## Make an amendment after crystallisation
 
-If a customer wants to make any changes following crystallisation they have 12 months from the statutory filing date to do this (the statutory filing date is 31 January following the end of the tax year, or 3 months from receipt of a Notice to File by the taxpayer whichever is the later). 
+If a customer wants to make any changes following crystallisation they have 12 months from the statutory filing date to do this (the statutory filing date is 31 January following the end of the tax year, or 3 months from receipt of a Notice to File by the taxpayer whichever is the later).
```

Comment:
Is taxpayer one or two words? (same comment as before)

Reply:
good call - it's one word (but two if its a council tax payer)